classytags/helpers.py

```diff
@@ -83,7 +84,10 @@ def render_tag(self, context, **kwargs):
             safe_context.update(**data)
             output = render_to_string(template, safe_context)
         else:
-            data = context.new(self.get_context(context, **kwargs))
+            new_context = context.new(
+                flatten_context(self.get_context(context, **kwargs))
+            )
+            data = flatten_context(new_context)
```

Comment:
Yes. When you switch to edit mode the same error appears. The error is that one of the context to be flattened is a list instead of a dict. Trying to track down where this wrong context is generated